haproxy/map.jinja

```diff
@@ -6,6 +6,9 @@
       'role': haproxy_role,
       'this': salt['grains.filter_by']({
         'Debian': {
+          'basic_auth_enable': False,
+          'basic_auth_user': 'admin',
+          'basic_auth_password': 'INVALID_USE_MKPASSWD_TO_CREATE',
```

Comment:
Good point, will update documentation and set default basic_auto_passwd to,
urm, 'SEE_README'?

On 30 January 2015 at 14:52, Ash Berlin <notifications@github.com> wrote:

> In haproxy/map.jinja
> <https://github.com/ministryofjustice/haproxy-formula/pull/6#discussion_r23846520>
> :
>
> > @@ -6,6 +6,9 @@
> >        'role': haproxy_role,
> >        'this': salt['grains.filter_by']({
> >          'Debian': {
> > +          'basic_auth_enable': False,
> > +          'basic_auth_user': 'admin',
> > +          'basic_auth_password': 'INVALID_USE_MKPASSWD_TO_CREATE',
>
> I'm of the opinion that if we can't provide a sensible default value then
> we shouldn't have something show up in the pillar.
>
> Though that said this is the closes thing we have to documentation for
> this feature so... Either we put a README in that documents how we do it or
> decide that this just about works.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/haproxy-formula/pull/6/files#r23846520>
> .
>